Include/AndroidExtensions/Globals.h

```diff
@@ -1,17 +1,25 @@
 #pragma once
 
 #include <jni.h>
+#include <android/asset_manager_jni.h>
 #include <arcana/containers/ticketed_collection.h>
 #include "JavaWrappers.h"
 
 namespace android::global
 {
     void Initialize(JavaVM* javaVM, jobject appContext);
```

Comment:
nit: This isn't introduced by your changes, but maybe good to fix with your changes.

```suggestion
    void Initialize(JavaVM* javaVM, jobject context);
```